sagemaker-python-sdk/mxnet_horovod_maskrcnn/source/requirements.txt

```diff
@@ -0,0 +1,5 @@
+gluoncv==0.8.0b20200723 
+pybind11==2.4.3
+Cython==0.29.6
+mpi4py==3.0.3
+pycocotools
```

Comment:
Are you able to install NVIDIA's COCO API? Here is how it should be installed

`pip --no-cache-dir --no-cache install 'git+https://github.com/NVIDIA/cocoapi#egg=pycocotools&subdirectory=PythonAPI'`

Reply:
Why do we need to do this?
```
pip install -r requirements.txt
```
successfully installs
```
 pycocotools-2.0.2
```

Reply:
Mask RCNN script depends on NVIDIA's pycocotools. Otherwise you might see this error:

`AttributeError: external module is not support by the COCO API installed. Consider install NVIDIA MSCOCO API here https://github.com/NVIDIA/cocoapi.git`